### PR TITLE
docs: Update the README to the latest ECP config schema.

### PR DESCRIPTION
**Pull request by clundin25**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,12 +97,13 @@ ECP relies on the `certificate_config.json` file to read all the metadata inform
       "token_label": "YOUR_TOKEN_LABEL",
```
Comment (andyrzhao) [inline README.md:97]: Can you update this based on the currently supported fields? It should be label, slot, and module I believe.

**Review by andyrzhao — CHANGES_REQUESTED** [review]
Looks great overall! Minor comment regarding pkcs11 config.

File: README.md (hunk 1 above)
Comment (clundin25) [reply README.md]: Good catch, thanks!

**Review by andyrzhao — APPROVED** [review]
LGTM

**Pull request merged**